src/Sylius/Bundle/ThemeBundle/Command/DebugThemeCommand.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Sylius\Bundle\ThemeBundle\Command;
+
+use Sylius\Bundle\ThemeBundle\Model\ThemeInterface;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+class DebugThemeCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('sylius:debug:theme')
```

Comment:
On the other hand, Symfony2 has recently changed their command naming from `subject:action` to `action:subject`, eg. `container:debug` became `debug:container`.

The same comes with service naming, I prefer `sylius.subject.service_name` to `sylius.service_name.subject`. For example, SyliusResourceBundle introduces `sylius.repository.resource` services, but even when I don't use it, I follow these rules just to be consistent.